dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ScalaClassLoaderManager.scala

```diff
@@ -62,7 +62,18 @@ class ScalaClassLoaderManager @Inject()(logger: MavenLoggerProxy) {
         classLoader
       case None =>
         logger.debug(s"ScalaClassLoader cache miss - $cacheKey")
-        val classLoader = new URLClassLoader(scalaArtifacts.map(_.getFile.toURI.toURL).toArray, null)
+        // Use System classloader parent as documented here:
+        // https://svn.apache.org/repos/infra/websites/production/maven/content/reference/maven-classloading.html#Maven_API_classloader
+        // Keep in mind this does not contain any contain any application or javaagent classes, which will
```

Comment:
```suggestion
        // Keep in mind this does not contain any application or javaagent classes, which will
```

Reply:
I cannot accept the suggestion here, so pushed a new commit.

Reply:
Yep, indeed. The suggestions can only be applied by the original PR owner.